### PR TITLE
[v7.3.x] Tracing: log traceID in request logger

### PR DESCRIPTION
**Pull request by papagian**
Backport db637a3630ffcc79ecb4b99bcf633d4563741833  from #28952

### CODE REVIEW

File: pkg/middleware/logger.go [hunk 1]
```diff
@@ -33,11 +36,11 @@ func Logger() macaron.Handler {
 		rw := res.(macaron.ResponseWriter)
 		c.Next()
 
-		timeTakenMs := time.Since(start) / time.Millisecond
+		timeTaken := time.Since(start) / time.Millisecond
```
Comment (bergquist) [inline pkg/middleware/logger.go:39]: Dont think I changed those but looks fine :)

**Review by bergquist — APPROVED** [review]
LGTM

File: pkg/middleware/logger.go (hunk 1 above)
Comment (papagian) [reply pkg/middleware/logger.go:39]: You didn't! this change was introduced in 3d3a7cbba80e02017455947ec7608a03a8c95e44

Comment (aknuds1) [reply pkg/middleware/logger.go:39]: I wonder how come it got included in this PR, but doesn't make any difference.

**Review by aknuds1 — APPROVED** [review]

**Pull request merged**